### DISCUSSION TITLE
Nothing happens when I run the .exe

### DISCUSSION THREAD

**Discussion by KodyGW** [kangyu-california/PersistentWindows#325] [Q&A]
When I run the .exe nothing happens. No popup, no crash, nothing in system tray or task manager, no files generated in appdata, nothing. I just see the loading icon on my mouse for about a second and that's it. I tried running as administrator and it didn't change anything. I'm on Windows 11. Are there any known conflicts with Open-Shell or RetroBar?

**Comment by kangyu-california** [comment]
workaround was provided in #318 
or run the latest patch
[PersistentWindows5.54_patch3.zip](https://github.com/user-attachments/files/15517082/PersistentWindows5.54_patch3.zip)
